### DISCUSSION TITLE
Hey

### DISCUSSION THREAD

**Discussion by Wishlah** [Repooc/ElvUI_DynamicStatusIcons#13] [Ideas]
I love these very much. Do you plan on adding the ability to choose an icon set for each class and one for NPCs? Maybe even different types of NPCs. Because that would be amazing.

Thanks!

**Comment by Repooc** [comment]
Nah, wasnt trying to get all convoluted with a bunch of options, and how to go about that in the settings seems like a mess with how many icons there will be... not saying I wont but def not planned... ideas on a how the config would be laid out may help but still doesnt mean i will if someone thinks of a good way to lay out such options

**Reply by Wishlah** [reply]
Yeah fair enough.

I wouldn't know anything about creating addons but if you're after ideas, I think the simplest config layout would be a list of tickboxes at the bottom or on a separate tab for all the classes/etc and to the right of each a simple drop down menu that lists the names of all the sets for you to choose from. Just the names of them rather than the images. People could check the names/browse them when choosing the default/fallback set with how that is atm.

Or no tickboxes just list the classes/whatever and have 'use default' as default dropdown entry.
